packages/prompts/package.json

```diff
@@ -55,7 +55,8 @@
   "dependencies": {
     "@clack/core": "workspace:*",
     "picocolors": "^1.0.0",
-    "sisteransi": "^1.0.5"
+    "sisteransi": "^1.0.5",
+    "std-env": "^3.4.3"
```

Comment:
this is an entire new package simply to check an environment variable. I've yet to find a CI platform which doesn't set an env var called CI=true.

If this library hopes to retain it's small size, it probably makes sense not to add this dependency, as I don't think it is required.